its/plugin/tests/src/test/java/com/sonar/it/php/PHPUnitTest.java

```diff
@@ -74,18 +74,36 @@ public void tests() throws Exception {
 
   @Test
   public void coverage() throws Exception {
-    assertThat(getProjectMeasure("lines_to_cover").getValue()).isEqualTo(6);
-    assertThat(getProjectMeasure("uncovered_lines").getValue()).isEqualTo(2);
+    assertThat(getProjectMeasure("lines_to_cover").getValue()).isEqualTo(21);
+    assertThat(getProjectMeasure("uncovered_lines").getValue()).isEqualTo(17);
     assertThat(getProjectMeasure("conditions_to_cover")).isNull();
     assertThat(getProjectMeasure("uncovered_conditions")).isNull();
 
+    assertThat(getCoveredFileMeasure("lines_to_cover").getValue()).isEqualTo(6);
+    assertThat(getCoveredFileMeasure("uncovered_lines").getValue()).isEqualTo(2);
+    assertThat(getCoveredFileMeasure("conditions_to_cover")).isNull();
+    assertThat(getCoveredFileMeasure("uncovered_conditions")).isNull();
+
+    assertThat(getUnCoveredFileMeasure("lines_to_cover").getValue()).isEqualTo(15);
+    assertThat(getUnCoveredFileMeasure("uncovered_lines").getValue()).isEqualTo(15);
+    assertThat(getUnCoveredFileMeasure("conditions_to_cover")).isNull();
+    assertThat(getUnCoveredFileMeasure("uncovered_conditions")).isNull();
+
     // see MMF-345
     if (is_before_sonar_6_2()) {
```

Comment:
@henryju I think it's worth to assert that we have zero coverage when SQ>6.1 and no reports are provided. We could simply add such assertion to PHPIntegrationTest